src/sprite.cpp

```diff
@@ -164,6 +167,17 @@ template <typename T> T* from_table(T* table, int level, int number) {
     const char* const* asar_prints = asar_getprints(&print_count);
     for (int i = 0; i < print_count; i++)
         io.debug("Asar print from %s: %s\n", file.path().c_str(), asar_prints[i]);
+
+    if (cfg.SymbolsType.length()) {
+        const char* symbols_contents = asar_getsymbolsfile(cfg.SymbolsType.c_str());
+        std::string symbols_path = file.path() + "." + cfg.SymbolsType;

```

Comment:
So here we end up writing a filename called `<filename>.asm.wla`.
I don't really like the double file extension.
I feel like this could be changed to
`std::string symbols_path{fs::path{file.path()}.replace_extension(cfg.SymbolsType).generic_string()};`
This way we avoid having .asm.wla and just .wla.